web/client/components/misc/datetimepicker/DateTimePicker.js

```diff
@@ -106,22 +106,69 @@ class DateTimePicker extends Component {
         const { date: dateFormat, time: timeFormat, base: defaultFormat } = formats;
         return format ? format : !time && calendar ? dateFormat : time && !calendar ? timeFormat : defaultFormat;
     }
+    renderCustomDateTimePopup = () => {
+        const { inputValue, operator, open } = this.state;
+        const { placeholder, tabIndex } = this.props;
 
-    renderInput = (inputValue, operator, toolTip, placeholder, tabIndex, calendarVisible, timeVisible) => {
+        const timeVisible = open === 'time';
+        const props = Object.keys(this.props).reduce((acc, key) => {
+            if (['placeholder', 'calendar', 'time', 'onChange', 'value'].includes(key)) {
+                // remove these props because they might have undesired effects to the subsequent components
+                return acc;
+            }
+            acc[key] = this.props[key];
+            return acc;
+        }, {});
```

Comment:
you can use "omit by" from lodash lib

or better if you which are the props you need you can use "pick"